demo/pages/index.module.css

```diff
@@ -0,0 +1,3 @@
+.svg {
```

Comment:
Same as before, check https://github.com/moxystudio/react-lib-template/pull/5 for reference.

```css
.home {
    & .svg {
        fill: #00aced;
    }
}
```